testing/admin/Admin004 - create team - field validations.md

```diff
@@ -6,16 +6,21 @@
 *Preconditions*
 * Admin user is logged in
 * Team list is opened
+* teamA is already created
 
 |Test Step|Expected Result|
 |---------|---------------|
 |Leave all fields empty.|All fields are empty.|
-|Save the team.|Team is not saved.<br>There are errors under the both fields displayed.|
+|Save the team.|Team is not saved.<br>There is a FE empty field validation displayed.|
```

Comment:
Trochu jsem se na to díval a googlovala  přemýšlím...
Chcete to rozdělovat v testech jako FE a BE validace? Tady jsou ty inputy celkem jednoduché, ale obecně ne všechny validace jsou podporované stejně ve všech prohlížečích asi.

Jestli nemít FE validaci a BE jako fallback, ale nerozlišovat to. Prostě systém musí vyhodit chybu, ale je jedno na které vrstvě.

Vůbec nevím, jak se tohle řeší, tak jestli k tomu máš nějaké info, budu rád.

Reply:
asi je to zbytecne rozlisovat :)
upravime testcasy a budeme overovat, ze tym nejde ulozi a chyba je zobrazena